src/typeahead/typeahead.ts

```diff
@@ -214,6 +214,12 @@ export class NgbTypeahead implements ControlValueAccessor,
     this._renderer.setProperty(this._elementRef.nativeElement, 'disabled', isDisabled);
   }
 
+  onDocumentClick(event) {
+    if (event.target !== this._viewContainerRef.element.nativeElement) {
```

Comment:
Shouldn't it be `this._elementRef. nativeElement`?

Reply:
Yes it should, my bad, fixing it now